### PR TITLE
Bump membrane core version to 1.0 for VideoCompositor SDK, flatten examples file structure

### PR DESCRIPTION
**Pull request by WojciechBarczynski**

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -42,12 +42,12 @@ defmodule Membrane.VideoCompositor.Mixfile do
   defp deps do
     [
       # Membrane
-      {:membrane_core, "~> 0.12.9"},
+      {:membrane_core, "~> 1.0.0"},
```
Comment (mat-hek) [inline mix.exs:45]:
```suggestion
      {:membrane_core, "~> 1.0"},
```

File: examples/mix.exs [hunk 2]
```diff
@@ -27,16 +27,16 @@ defmodule Membrane.VideoCompositor.Examples.Mixfile do
   defp deps do
     [
       # Membrane
-      {:membrane_core, "~> 0.12.9"},
+      {:membrane_core, "~> 1.0.0"},
```
Comment (mat-hek) [inline examples/mix.exs:30]:
```suggestion
      {:membrane_core, "~> 1.0"},
```

File: examples/lib/dynamic_outputs.exs [hunk 3]
```diff
@@ -1,7 +1,8 @@
-alias Membrane.VideoCompositor.Examples.DynamicOutputs.Pipeline
+alias Membrane.VideoCompositor.Examples.DynamicOutputs.Pipeline, as: DynamicOutputsPipeline
```
Comment (mat-hek) [inline examples/lib/dynamic_outputs.exs:1]: Didn't notice that before, but these names can awaken Java PTSD. Let's rename `Membrane.VideoCompositor.Examples.DynamicOutputs.Pipeline` to `DynamicOutputs.Pipeline` or `DynamicOutputsPipeline` and don't alias at all

**Review by mat-hek — CHANGES_REQUESTED** [review]

File: examples/lib/dynamic_outputs.exs (hunk 3 above)
Comment (mat-hek) [reply examples/lib/dynamic_outputs.exs:1]: And you can move the contents of `dynamic_outputs/pipeline.ex` into this script, I don't see a benefit for having this split into multiple files

File: examples/lib/dynamic_outputs.exs [hunk 4]
```diff
@@ -1,7 +1,202 @@
-alias Membrane.VideoCompositor.Examples.DynamicOutputs.Pipeline
+defmodule DynamicOutputsPipeline do
```
Comment (WojciechBarczynski) [inline examples/lib/dynamic_outputs.exs:1]: Everything here is copy-pasted from deleted files

**Review by mat-hek — COMMENTED** [review]
ok, now you can move the .exs files out of the lib directory and fix readme https://github.com/membraneframework/membrane_video_compositor_plugin/tree/bump_membrane_core_version#examples and I think we can merge ;)

**Review by mat-hek — APPROVED** [review]

**Pull request merged**